packages/@react-aria/focus/docs/FocusRing.mdx

```diff
@@ -35,8 +35,10 @@ keywords: [focus, keyboard focus, focus management, aria]
 
 `FocusRing` is a utility component that can be used to apply a CSS class when an element has keyboard focus.
 This helps keyboard users determine which element on a page or in an application has keyboard focus as they
-navigate around. Focus rings are only visible when interacting with a keyboard so as not to distract mouse
-and touch screen users.
+navigate around. Focus rings are usually suppresed for mouse and touch screen users, unless browser tabs are
+switched away and back, or while navigating a form with the next/previous buttons on iOS.
+
+Cha
```

Comment:
Cha?

I think we want to keep the information about why the focus rings are suppressed for mouse/touch users.
Maybe we leave it as it was and add a sentence saying `In cases where we are unable to detect if the user is using a mouse or touch screen, such as switching in from a different tab, we show the focus ring.`

Reply:
"suppresed" is spelled incorrectly in submitted PR.

Reply:
yes, good suggestion!

I was unsure of the wording.